src/test/java/net/snowflake/ingest/TestUtils.java

```diff
@@ -302,6 +302,7 @@ public static Connection getConnection(boolean isStreamingConnection) throws Exc
     props.put("warehouse", warehouse);
     props.put("client_session_keep_alive", "true");
     props.put("privateKey", privateKey);
+    props.put("role", role);
 
```

Comment:
i'm guessing you've updated the role in the files to be SYSADMIN too..
Did that not require any special permissions in any environment, for instance to grant usage on EXT VOL to role SYSADMIN?

Reply:
Yes I've grantde permission on those sfctest0 accounts.